### PR TITLE
feat: squads notification strip

### PR DESCRIPTION
**Pull request by el**
## Changes

Screenshot 2023-01-18 at 17 11 36

### Describe what this PR does
- Short and concise, bullet points can help
- Screenshots if applicable can also help

## Events

Did you introduce any new tracking events?
Don't forget to update the [Analytics Taxonomy sheet](https://docs.google.com/spreadsheets/d/18Lv7zXges9QfVX5VYL1a-Hyl0e1sQ3sLr0OK8YZWKXI/edit#gid=0)

Log the new/changed events below:

| Type   | event_name  | value |
|--------|-------------|-------|
| Change/New | event name  | extra: { ... } |

### On those affected packages:
- [ ] Have you done sanity checks in the webapp?
- [ ] Have you done sanity checks in the extension?
- [ ] Does this not break anything in companion?

### Did you test the modified components media queries?
- [ ] MobileL (420px)
- [ ] Tablet (656px)
- [ ] Laptop (1020px)

#### Did you test on actual mobile devices?
- [ ] iOS (Chrome and Safari)
- [ ] Android

WT-{number} #done

### CODE REVIEW

File: packages/shared/src/components/comments/MainComment.tsx [hunk 1]
```diff
@@ -43,6 +45,7 @@ export default function MainComment({
         <EnableNotification
           className={!comment.children?.edges?.length && 'mt-3'}
           source={NotificationPromptSource.NewComment}
+          contentName={user?.id !== comment?.author.id ? comment?.author?.name : undefined}
```
Comment (el) [inline packages/shared/src/components/comments/MainComment.tsx:48]: @sshanzel I think this was removed by mistake before

Comment (el) [inline packages/shared/src/components/comments/MainComment.tsx]: @sshanzel I think this was removed by mistake before

**Review by rebelchris — APPROVED** [review]
Love it! 🙏

Just make double sure with @sshanzel what happened with that comment removal.

File: packages/shared/src/components/comments/MainComment.tsx (hunk 1 above)
Comment (sshanzel) [reply packages/shared/src/components/comments/MainComment.tsx]: I don't really remember anything related to this. Could have gotten lost along the way by mistake through merge. Apologies for that as I was the one who did it. Will carefully merge bigger things on the next ones 🙏

**Review by sshanzel — APPROVED** [review]
Just one thing with regards to the discussion I had Sab about copy. Other than that, everything looks amazing! 🚢

File: packages/shared/src/components/notifications/EnableNotification.tsx [hunk 2]
```diff
@@ -104,13 +105,14 @@ function EnableNotification({
...
     [NotificationPromptSource.NewComment]: `Want to get notified when ${
-      parentCommentAuthorName ?? 'someone'
+      contentName ?? 'someone'
     } responds so you can continue the conversation?`,
     [NotificationPromptSource.CommunityPicks]:
       'Would you like to get notified on the status of your article submissions in real time?',
     [NotificationPromptSource.NotificationsPage]:
       'Stay in the loop whenever you get a mention, reply and other important updates.',
     [NotificationPromptSource.NewSourceModal]: '',
+    [NotificationPromptSource.SquadPage]: `Get notified whenever something important happens on "${contentName}".`,
```
[the remark this answers is not in the document]
Comment (sshanzel) [reply packages/shared/src/components/notifications/EnableNotification.tsx:115]: Discussion quick link: https://dailydotdev.slack.com/archives/C03QZBF46JC/p1673625112519489

**Pull request merged**